examples/react-native/README.md

```diff
@@ -84,6 +84,32 @@ yarn react-native-example dev
 
 1. Open the app on the iOS simulator or Android emulator.
 
+## Storybook
+
+First boot Storybook Server, which will serve a web interface that allows switching between components (stories) and controlling multiple simulators/emulators:
+
+```bash
+# terminal 1
+yarn run react-native-example storybook
+```
+
+From a second terminal run ios simulator, android emulator or both:
+
+```bash
+# terminal 2
```

Comment:
It would be really helpful to not have to run commands in a second terminal, and run the web storybook simulator optionally. Is there an approach we can take that would allow that?

Reply:
Yes running Storyboook is optional, though I discourage it, ideally we should test these in multiple environments but I will add notes.

Reply:
Is running the storybook without the web server and testing the components on Android and iOS different then run testing on multiple platforms with the web server? Seems like with the web server the major difference is that it controls the emulator and/or simulator, am I missing something else?

Reply:
Correct, Storybook server controls all emulators/simulators connected to it, that is the key difference. The Storybook app itself has a tab menu and you can freely switch between the Navigator and Preview tab (see updated PR description) to access the stories - more clicks though :)